docs/tutorials/use.md

```diff
@@ -131,14 +81,7 @@ localhost/podenv/firefox                    latest                622df31ab30b
 $ podman tag 713d6fa70a9f localhost/podenv/firefox
```

Comment:
Would be nice to have podenv propose an option for that instead of looking/tagging for the previous image manually

Reply:
ditto.